app/renderer/components/common/switchControl.js

```diff
@@ -131,7 +131,8 @@ class SwitchControl extends ImmutableComponent {
               styles.switchControl__text__label,
               styles.switchControl__text__label_right,
               this.props.small && styles.switchControl__text__label_small,
-              this.props.customStyleTextRight
+              this.props.customStyleTextRight,
+              this.props.disabled && styles.switchControl__text_disabled
```

Comment:
@user512 would you mind moving `this.props.customStyleTextRight` under this line as it is expected to override the inherited properties?